src/DIRAC/ConfigurationSystem/Agent/RucioSynchronizerAgent.py

```diff
@@ -104,42 +125,195 @@ def getStorageElements():
             params['domains']['wan']['delete'] = write_protocols.get(value, 0)
             params['domains']['wan']['third_party_copy'] = write_protocols.get(value, 0)
         seProtocols[seName].append(params)
+  log.debug("Accepted Dirac SEs: ", seProtocols)
   return S_OK(seProtocols)
 
 
+def configHelper(voList):
```

Comment:
Looks to me like this one would better fit in DIRAC/ConfigurationSystem/Client/Helpers/Resources.py (and there might be duplications)